### PR TITLE
The first module of Telefonica cloud

### PR DESCRIPTION
**Pull request by zengchen1024**
SUMMARY
This pr wants to add the first module of Telefonica cloud.

ISSUE TYPE
Feature Pull Request

### CODE REVIEW

File: test/integration/cloud-config-hwc.yml.template [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This is the configuration template for ansible-test HWC integration tests.
```
Comment (mattclay) [inline test/integration/cloud-config-hwc.yml.template]: This file is not needed since there is no cloud test plugin using it.

Comment (zengchen1024) [reply test/integration/cloud-config-hwc.yml.template]: thanks, I will remove it!

File: test/units/module_utils/hwc/test_dict_comparison.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+
+class HwcDictComparisonTestCase(unittest.TestCase):
+    def test_simple_no_difference(self):
+        value1 = {
+            'foo': 'bar',
+            'test': 'original'
+        }
+        d = DictComparison(value1)
+        self.assertTrue(d == d)
```
Comment (mattclay) [inline test/units/module_utils/hwc/test_dict_comparison.py]: Use `self.assertEqual(d, d)` instead of `self.assertTrue(d == d)`.

Comment (zengchen1024) [reply test/units/module_utils/hwc/test_dict_comparison.py]: @mattclay Thanks for your reviewing!  I have done the modification. Please review again. Thanks!

**Comment by zengchen1024** [comment]
@mattclay Could you please take a look at this pr? Thank you very much!!!

**Comment by Click2Cloud-Gamma** [comment]
@mattclay It would be really great if you can share your feedback regarding the PR OR please review it once again?

**Comment by Click2Cloud-Gamma** [comment]
bot_status

**Comment by Click2Cloud-Gamma** [comment]
@ansibot @mattclay @samdoran The PR is ready_for_review. Could you please review it.

**Comment by Click2Cloud-Gamma** [comment]
@ansibot @mattclay @samdoran Please have a look at this PR and review it once again. Thank you so much.
ready_for_review

**Comment by Click2Cloud-Gamma** [comment]
@ansibot @mattclay Could you please review this PR once again for further process. Thank you so much.
ready_for_review

**Comment by Click2Cloud-Gamma** [comment]
@ansibot @mattclay @samdoran The PR is ready_for_review. Could you please review it.

**Comment by zengchen1024** [comment]
@ansibot @mattclay @samdoran The PR is ready_for_review. Could you please review it.

**Comment by Click2Cloud-Gamma** [comment]
@ansibot @mattclay @samdoran Please have a look at this PR and review it once again. 
ready_for_review

**Pull request closed without merging**